src/quo2/components/drawers/drawer_buttons/style.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.drawers.drawer-buttons.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def outer-container {:height 216})
+
+(def top-card
+  {:height             "100%"
+   :display            :flex
```

Comment:
no need to specify "100%" and :flex

Reply:
actually it's needed here, the card doesn't display otherwise 🤔